autorest/version.go

```diff
@@ -18,11 +18,14 @@ var version string
 // Version returns the semantic version (see http://semver.org).
 func Version() string {
```

Comment:
@marstr I modified the code as you suggested. The version is set only once if is empty.